src/app/clusters/joint-fabric-administrator-server/joint-fabric-administrator-server.cpp

```diff
@@ -126,15 +118,69 @@ bool emberAfJointFabricAdministratorClusterAddICACCallback(
     return true;
 }
 
-// TODO
 bool emberAfJointFabricAdministratorClusterOpenJointCommissioningWindowCallback(
     chip::app::CommandHandler * commandObj, const chip::app::ConcreteCommandPath & commandPath,
     const chip::app::Clusters::JointFabricAdministrator::Commands::OpenJointCommissioningWindow::DecodableType & commandData)
 {
+#if CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC
```

Comment:
Under what conditions would this file be compiled but CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC be false?

Reply:
Yes, I rebased the code.